utils/utils_test.ts

```diff
@@ -4,6 +4,7 @@ import { directorySize } from "./utils.ts";
 import type { DirectoryListingFile } from "./types.ts";
 
 Deno.test({
+  ignore: true,
```

Comment:
Why?

Reply:
oh, just forgot to remove this line before opening the PR